composer.json

```diff
@@ -50,8 +53,7 @@
         "behat/mink-goutte-driver": "^1.2",
         "jcalderonzumba/gastonjs": "^1.0.2",
         "jcalderonzumba/mink-phantomjs-driver": "^0.3.1",
-        "drupal/drupal-extension": "^3.1",
-        "phpunit/phpunit": "^4.8",
+        "drupal/drupal-extension": "^4.0",
         "symfony/css-selector": "^2.8"
     },
```

Comment:
several of these dev dependencies are actually causing some pinnned versions of libraries and some warnings about duplicate class resolution. If you limit this to just 
```
        "drupal/core-dev": "^9",
        "drupal/twig_xdebug": "^1.2",
        "phpcompatibility/php-compatibility": "^9.3"
```
we should have everything we need to be able to develop since we dont' have behat testing